home/templates/home/round_page_with_communities.html

```diff
@@ -135,7 +135,7 @@ <h3>Final Applications Closed {{ current_round.contributions_close }} at 4pm UTC
 
 	{% for participation, projects in ontime_projects %}
 	{% with community=participation.community funded_interns=participation.interns_funded %}
-		<h4 id='{{ community.slug }}'>{{ community.name }} - {{ funded_interns }} intern{{ funded_interns|pluralize }}</h4>
+	<h4 id='{{ community.slug }}'><a href="#{{ community.name|lower }}">{{ community.name }}</a> - {{ funded_interns }} intern{{ funded_interns|pluralize }}</h4>
```

Comment:
Hi @tapaswenipathak! Good idea using a template filter here, but the `name` field may not work if the community name has more than one word. I think using the `slug` as in the id should work perfectly.

Reply:
Hi @krysal! added. :)